src/components/base/outline-breadcrumbs/outline-breadcrumbs.ts

```diff
@@ -0,0 +1,30 @@
+// Our base component, which all others extend.
+import { OutlineElement } from '../outline-element/outline-element';
+import { CSSResultGroup, html, TemplateResult } from 'lit';
+import { customElement } from 'lit/decorators.js';
+import componentStyles from './outline-breadcrumbs.css.lit';
+import { SlottedController } from '../../controllers/slotted-controller';
```

Comment:
@sarahp `slotted-controller` is missing

Reply:
@shaal does the component need to be wrapped in a `<slotted-controller>`?

Reply:
@LeeMellon Without the slotted-controller, I couldn't style the individual links of the Drupal output.

Reply:
@sarahp I get that. I was wondering what @shaal was saying was missing.

Reply:
Oh NM now I see. You're importing it but it isn't there.

Reply:
It's in a gist because I thought Jake might be using it's own PR, but I could push it up if that's not the case: https://gist.github.com/sarahp/f23c222e8a01c214b7a95d5a249c41d1